mirdata/utils.py

```diff
@@ -184,6 +177,17 @@ def update_to(self, b=1, bsize=1, tsize=None):
         self.update(b * bsize - self.n)
 
 
+def download_large_file(url, download_path, callback=None):
```

Comment:
is a "large file" treated any differently than a small file? If not, I'd just call this function "download_file"

Reply:
It is!  `download_file_in_chunks_ would be the correct version - can change if you want?